charts/velero/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.6.0
 description: A Helm chart for velero
 name: velero
-version: 2.21.0
+version: 2.21.1
```

Comment:
I don't think we need to bump the Chart version since it's a CI enhancement.

Reply:
I was trying to remember if we had to or not. Will revert.